admin/ownership_grants/V1.15.1__synapse_event_ownership.sql

```diff
@@ -0,0 +1,22 @@
+-- SYNAPSE_DATA_WAREHOUSE
+GRANT OWNERSHIP
+    ON SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE_EVENT
+    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE.SYNAPSE_EVENT_ALL_ADMIN
+    REVOKE CURRENT GRANTS;
```

Comment:
What is the reason for using `REVOKE CURRENT GRANTS` here? Just wondering because I didn't see it when `SYNAPSE_AGGREGATE` was introduced

<img width="436" alt="image" src="https://github.com/user-attachments/assets/0c2fcf15-e1fc-4572-8ff9-aa24aaf2bcb2" />

Also, we are missing USAGE/MONITOR grants to `*ALL_DEVELOPER`